api_definition/coepi_0.2.0.yml

```diff
@@ -0,0 +1,144 @@
+openapi: 3.0.2
+info:
+  description: This is the API model for CoEpi Server
+  version: 0.2.0
+  title: CoEpi Server
+  license:
+    name: MIT License
+    url: https://github.com/Co-Epi/data-models/blob/master/LICENSE
+    
+servers:
+  - url: https://coepi.wolk.com:8080
+    description: URL Endpoint used for dev/test
+    
+paths:
+  /cenreport:
+    post:
+      summary: Submit symptom or infection report
+      description: Users submit symptom / infection reports and reveal the secret CENKey in their application
+      
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                reportId: # Suggestion: Why is this being passed by the client? Better to be auto-generated by server?
```

Comment:
That's what I think. But maybe I don't understand the purpose of this identifier. Sourabh mentioned this would not be a mandatory field. Let's wait for his input on this.